tests/microos/disk_boot.pm

```diff
@@ -28,6 +28,24 @@ sub run {
         shift->wait_boot(bootloader_time => 300);
     }
     microos_login;
+    record_kernel_audit_messages();
+}
+
+
+=head2 record_kernel_audit_messages
+
+Record the SELinux messages before the auditd daemon has been started, if present. If there are no such entries, this function has no effect.
```

Comment:
old SLEM images don't have SELinux enabled by default, so maybe we could run this module `if get_var('ENABLE_SELINUX')` ?

Reply:
You may find the messages even there https://openqa.suse.de/tests/10157368#step/install_updates/41

Reply:
@jlausuch But we are not using any `SELINUX` variables in many test runs, e.g. the verification run.